algebird-test/src/test/scala/com/twitter/algebird/BatchedTest.scala

```diff
@@ -52,6 +52,30 @@ class BatchedLaws extends CheckProperties {
   testBatchedMonoid[String]("String", 10)
   testBatchedMonoid[String]("String", 100)
   testBatchedMonoid[String]("String", 1000000)
+
+  def testBatchedSemigroup[A: Arbitrary: Semigroup](name: String, size: Int): Unit = {
```

Comment:
why was this added here. Seems a weird way to test bigdecimal?

Can we instead add a file like `NumericAlgebraTests.scala` that has something like:
```scala
class NumericAlgebraTests extends CheckProperties {
  property("test bigdecimal") {
    // check resolution:
    implicitly[Semigroup[BigDecimal]]
    implicitly[Monoid[BigDecimal]]
    implicitly[Group[BigDecimal]]
    // now test all the laws:
    ringLaws[BigDecimal]
  }
```
and similar for the numeric types we care about?

Reply:
yeah... I wanted to add _something_ as a test... even if it was crude... I'll add this instead